source/configuration_manual/service_configuration.rst

```diff
@@ -302,7 +302,7 @@ Settings
    to distinguish different listener types that one service may employ.
 
 
-.. dovecot_core:setting:: inet_listener_address
+.. dovecot_core:setting:: inet_listener_listen
    :values: @ip_addresses
 
    Overrides the :dovecot_core:ref:`listen` setting for this listener.
```

Comment:
There is no such setting anymore. There is only the global listen setting. So remove this entirely. But you could update the global listen setting to say it can be overridden for inet_listeners and show an example of it.